packages/astro/src/@types/astro.ts

```diff
@@ -545,10 +545,12 @@ export interface AstroUserConfig {
 	 * ```
 	 *
 	 * When using this option, all of your static asset imports and URLs should add the base as a prefix. You can access this value via `import.meta.env.BASE_URL`.
+	 * 
+	 * By default, the value of `import.meta.env.BASE_URL` includes a trailing slash. If you have [`trailingSlash`](/en/reference/configuration-reference/#trailingslash) set to `'never'`, then you will need to add the trailing slash manually in your static asset imports and URLs
```

Comment:
I'm not sure relative links will work accordingly here, from what I saw in the same file we're mostly using absolute links, so to be sure, I'd change that. I also did a few changes in the writing, let me know what you think!
````suggestion
	 * By default, the value of `import.meta.env.BASE_URL` includes a trailing slash. If you have the [`trailingSlash`](https://docs.astro.build/en/reference/configuration-reference/#trailingslash) option set to `'never'`, you will need to add it manually in your static asset imports and URLs.
````

Reply:
Yes, I think @Yan-Thomas is right here — will add his suggestion 👍